src/database/models.py

```diff
@@ -458,16 +458,18 @@ class Community(models.Model):
       about this community
     more_info: JSON
       any another dynamic information we would like to store about this location
+
+    postmark_contact_info: JSON
+       looks like this {"is_validated":bool , "is_nudged":bool ,"sender_signature_id":str}
     """
 
     id = models.AutoField(primary_key=True)
     name = models.CharField(max_length=SHORT_STR_LEN)
     subdomain = models.SlugField(max_length=SHORT_STR_LEN, unique=True, db_index=True)
     owner_name = models.CharField(max_length=SHORT_STR_LEN, default="Unknown")
     owner_email = models.EmailField(blank=False)
-    owner_phone_number = models.CharField(
-        blank=True, null=True, max_length=SHORT_STR_LEN
-    )
+    sender_signature_name = models.CharField(blank=True, null=True, max_length=SHORT_STR_LEN)
+    owner_phone_number = models.CharField(blank=True, null=True, max_length=SHORT_STR_LEN)
     about_community = models.TextField(max_length=LONG_STR_LEN, blank=True)
```

Comment:
Change owner_name to contact_name, owner_email to contact_email, owner_phone_number to contact_phone_number.  Because there is no 'owner', this is really the contact person.
Also change sender_signature_name to contact_sender_alias (or put in the contact_info json field)
Also change postmark_contact_info to contact_info since I don't think 'postmark' needs to be in the model name.